src/packagedcode/models.py

```diff
@@ -117,14 +117,6 @@ class AssertedLicense(Model):
     notice = StringType(default=None)
     url = StringType(default=None)
 
-    def as_dict(self):
```

Comment:
Related we should not depend on the default constructor for schematics models to pass a dict vs named arguments: we should instead abstract that and accept **kwargs and pass the kwargs to schematics